testsuite/features/support/env.rb

```diff
@@ -58,6 +58,7 @@
 $is_using_build_image = ENV.fetch('IS_USING_BUILD_IMAGE', false)
 $is_using_scc_repositories = (ENV.fetch('IS_USING_SCC_REPOSITORIES', 'False') != 'False')
 $catch_timeout_message = (ENV.fetch('CATCH_TIMEOUT_MESSAGE', 'False') == 'True')
+$beta_enabled = ENV.fetch('BETA_ENABLED', false)
```

Comment:
We need to keep in mind using this in sumaform/susemanager-ci once we have beta channels again.

Reply:
Yup. Or at least, change the default to true.